<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWindowAdminstrator</class>
 <widget class="QMainWindow" name="SystemWindowAdminstrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>10</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Courses Requests</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableCourseRequests">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>541</width>
       <height>251</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Course Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Department</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Professor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>311</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelUserAction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Users</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="userTypesWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayoutUserType">
      <item>
       <widget class="QRadioButton" name="rbAdministrator">
        <property name="text">
         <string>Administrator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbProfessor">
        <property name="text">
         <string>Professor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStudent">
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Student</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pbAction">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leFirstName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLastName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Department</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbDepartmentUser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>94</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Date of Birth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditDateOfBirth">
         <property name="displayFormat">
          <string>dd-MM-yyyy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Start Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbStartYear"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Start Term</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbStartTerm"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>135</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Middle Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leMiddleName"/>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>pbAction</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>layoutWidget_5</zorder>
     <zorder>layoutWidget_6</zorder>
     <zorder>layoutWidget_7</zorder>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbCreate">
        <property name="text">
         <string>Create</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>9</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User Actions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="userListWidget" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayoutUserList">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>User List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbUserList"/>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>541</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Departments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_departments">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>141</width>
       <height>47</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbDepartmentAdmin"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Privileges</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbConfirmPriviliges">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>110</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Confrim</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>47</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Administrator Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAdminUsername"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbLogout">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>530</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
